hathor/p2p/peer_id.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from hathor.types import Hash
+
+
+class PeerId(Hash):
+    pass
```

Comment:
Couldn't you use a `NewType` instead?

Reply:
I guess it would work. It seems to have the exact same effect. Initially I expected that I might need custom methods. But anyway I'm only moving this from `hathor.p2p.entrypoint` to a new module `hathor.p2p.peer_id`, so I didn't even think of changing it.

Reply:
Maybe it would fit better into a `hathor.p2p.types` module, but I think we can structure this later when we have more of these types.

Reply:
Ok!